### PR TITLE
dep: update snyk to 1.1012.0

### PR DESCRIPTION
**Pull request by eunomie**
**- What I did**

Update snyk to 1.1012.0

**- How I did it**

**- How to verify it**

**- Description for the changelog**

Update snyk version to be used to 1.1012.0.

**- A picture of a cute animal (not mandatory)**

### CODE REVIEW

File: e2e/version_test.go [hunk 1]
```diff
@@ -123,5 +123,8 @@ Provider:   %s
 }
 
 func getProviderVersion(env string) string {
-	return fmt.Sprintf("Snyk (%s (standalone))", os.Getenv(env))
+	if runtime.GOOS != "windows" {
```
Comment (mcapell) [inline e2e/version_test.go:126]: question: Why is this needed?

**Review by mcapell — APPROVED** [review]
LGTM!

File: e2e/version_test.go (hunk 1 above)
Comment (eunomie) [reply e2e/version_test.go:126]: I'm not entirely sure what changed (but I validated it on a windows machine) but on windows machine, the standalone version doesn't print anymore ` (standalone)` in the version, only on windows.
It's not a big issue, what we want to know here is we got the right version number.
